uber/sep_commands.py

```diff
@@ -14,19 +14,10 @@ def print_config():
 @entry_point
 def insert_admin():
     with Session() as session:
```

Comment:
Oh wait, I just realized you're passing in the session to the function.

In general, whenever we have a function which takes a session object as its first argument, we typically put that in the ``SessionMixin`` class in ``models.py`` so that we can call it as a session method.

Alternatively, you could just move the session instantiation into the utility function itself.

Not a huge deal, since this will work as-is, it'sj ust not how we're doing things elsewhere in the codebase.